generate.py

```diff
@@ -0,0 +1,28 @@
+import yaml
+from typing import Mapping
+from terraform_templates import iam_template
+
+
+# generates all the necessary files to deploy policies given a configuration file in the format specified in /config.yml.template
+class Generator:
```

Comment:
I would consider a more descriptive name like `ConfigGenerator` or `PolicyGenerator`, both for clarity and to avoid conflicting with `from typing import Generator`.

Reply:
Good call, fixed.